provisioning/roles/kuma/files/apache2/all-servers.conf

```diff
@@ -3,16 +3,12 @@
 # Our servers symlink to this file. Do not move it without first coordinating
 # with WebOps.
 
-<Location /media/fonts>
+<Location /static/fonts>
     # Only enable CORS for fonts when the request is coming from a Mozilla domain
     Header unset Access-Control-Allow-Origin
     SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0
     SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
     Header set Access-Control-Allow-Origin %{CORS}e env=CORS
-
-    # block hotlinking by referer to .woff, .woff2, .eof, .ttf files except mozilla domains
-    RewriteCond "%{HTTP_REFERER}" "!https?://.*\.(mozilla|allizom)\.(com|org|net)/.*$"
-    RewriteRule \.(woff|woff2|eot|ttf)$ - [F,NC,L,E=!CORS]
```

Comment:
+1 for this. I think it was the last thing blocking merge? (Though now this needs another rebase)